src/usr/local/emhttp/plugins/tailscale/include/Pages/Dashboard.php

```
@@ -2,8 +2,15 @@
 
 namespace Tailscale;
 
+use EDACerton\PluginUtils\Translator;
+
+if ( ! defined(__NAMESPACE__ . '\PLUGIN_ROOT') || ! defined(__NAMESPACE__ . '\PLUGIN_NAME')) {
+    throw new \RuntimeException("Common file not loaded.");
+}
+
+$tr = $tr ?? new Translator(PLUGIN_ROOT);
```

PHPStan (2.1.15)

Instantiated class EDACerton\PluginUtils\Translator not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)